app/controllers/messages_controller.rb

```diff
@@ -146,12 +146,12 @@ def quote
     content = "#{ll(Setting.default_language, :text_user_wrote, user)}\\n> "
     content << text.to_s.strip.gsub(%r{<pre>((.|\s)*?)</pre>}m, '[...]').gsub('"', '\"').gsub(/(\r?\n|\r\n?)/, '\\n> ') + '\\n\\n'
     render(:update) { |page|
-      page << "$('message_subject').value = \"#{subject}\";"
-      page.<< "$('message_content').value = \"#{content}\";"
+      page << "$('reply_subject').value = \"#{subject}\";"
+      page.<< "$('reply_content').value = \"#{content}\";"
       page.show 'reply'
```

Comment:
@machisuji I think that problem is solved with https://travis-ci.org/opf/openproject/builds/59219347 failing across the board ^^